src/main/resources/templates/aeolus/ruby/default.sh

```
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -e
+
+test () {
+  echo '⚙️ executing test'
+  cd "${testWorkingDirectory}"
```

Shellcheck (0.10.0)

[warning] testWorkingDirectory is referenced but not assigned.

(SC2154)